internal/wasm/autocomplete.go

```diff
@@ -0,0 +1,40 @@
+package wasm
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+type AutoCompleteConfig struct {
+	JWT          string   `js:"jwt"`
+	LeftWords    []string `js:"leftWords"`
+	SelectedWord string   `js:"selectedWord"`
+	RightWords   []string `js:"rightWords"`
+}
+
+func Autocomplete(cfg *AutoCompleteConfig) ([]string, error) {
+	indexToComplete := int64(len(cfg.LeftWords) + 2)
+
+	words := append(cfg.LeftWords, cfg.SelectedWord)
+	words = append(words, cfg.RightWords...)
+
+	completeCommand := []string{"autocomplete", "complete", "zsh", strconv.FormatInt(indexToComplete, 10), "scw"}
```

Comment:
Could ZSH be in the AutoCompleteConfig? Maybe depending on the different feature of a shell we want to configure them differently?

Reply:
Could you add a comment about this directly in the code to give directions about how to refactor this code if you think it is going to be complex and time-consuming?